ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -124,6 +129,9 @@ function WorkloadCvesOverviewPage() {
                     setDefaultWatchedImageName('');
                     watchedImagesModalToggle.closeSelect();
                 }}
+                onWatchedImagesChange={() =>
+                    apolloClient.refetchQueries({ include: [imageListQuery] })
```

Comment:
When an image is added or removed from the watch list, we will refresh the`imageListQuery` if it is active.